packages/cozy-client/src/models/triggers.js

```diff
@@ -19,6 +37,49 @@ const triggers = {
       const message = JSON.parse(atob(trigger.message.Data))
       return message.konnector
     }
+  },
+  /**
+   * getAccountId - Returns the account id for a trigger
+   *
+   * @param {object} trigger io.cozy.triggers
+   *
+   * @returns {string} Id for an io.cozy.accounts
+   */
+  getAccountId: trigger => {
+    const legacyData = get(trigger, 'message.Data')
+
+    if (legacyData) {
+      const message = JSON.parse(atob(legacyData))
+      return message.account
+    } else {
+      return get(trigger, 'message.account')
+    }
+  },
+  /**
+   * isCurrentErrorMuted - Checks if the triggers current error has been muted in the corresponding io.cozy.accounts
+   *
+   * @param {object} trigger      io.cozy.triggers
+   * @param {object} accountsById Object where the keys are account ids, and the values are io.cozy.accounts
+   *
+   * @returns {boolean} Whether the error is muted or not
+   */
+  isCurrentErrorMuted: (trigger, accountsById) => {
```

Comment:
I found this method not speaking enough. 

For a `isCurrentErrorMuted` method, I would expect something like: 
`isCurrentErrorMuted: (error, trigger)`

In fact it's more a : `isLatestErrorMuted`

I don't like the `accountsById` attributes, is it something that is returned by a call to the stack? Why not just passing directly the `account` ?

Reply:
👍 for `isLatestErrorMuted `. For the accounts… it's not something returned by the stack, but it os a left over from the first implementation, and in hindsight you're 💯, the function should receive the account as a param.